docs/release.rst

```diff
@@ -65,6 +65,13 @@ Upcoming Release
 Bug fixes
 ~~~~~~~~~
 
+* Fix Sqlite Store Wrong Modification.
+  By :user:`potter420 <potter420>`; :issue:`440`.
```

Comment:
@potter420, are you ok being listed here? How would you like to be referred to?

Reply:
I would like to be referred to as Tommy Tran. 
My current name is kinda goofy.

Reply:
Sounds good. Thanks. Have pushed in the latest commit. Can you please let me know whether I got that correct?